packages/vx-brush/src/types.ts

```diff
@@ -19,13 +19,16 @@ export interface MarginShape {
   bottom?: number;
 }
 
-export interface BrushShape {
-  start: Point;
-  end: Point;
+export interface BrushShape extends ScrubberShape {
   extent: Bounds;
   bounds: Bounds;
 }
 
+export interface ScrubberShape {
```

Comment:
let's not introduce new naming semantics for the `Brush`, maybe we can be very literal and boring and simply call this `BrushStartEnd`. otherwise everyone will be confused why it's called brush sometimes and scrubber sometimes.

Reply:
Good idea! Thanks